internal/cli/organizations.go

```diff
@@ -553,8 +566,9 @@ func listRolesOrganizationCmd(cli *cli) *cobra.Command {
 		Args:    cobra.MaximumNArgs(1),
 		Short:   "List roles of an organization",
 		Long:    "List roles assigned to members of an organization.",
-		Example: `auth0 orgs roles list
-auth0 orgs roles ls <id>`,
+		Example: `  auth0 orgs roles list
+  auth0 orgs roles ls <id>
+  auth0 orgs roles ls <id> --json`,
```

Comment:
As an org is likely to have many roles assigned to its members, consider in a future PR adding an example with the `-n` flag, like in other similar commands.